code/client/clrcore/External/WeaponComponentCollection.cs

```diff
@@ -17,319 +17,434 @@ namespace CitizenFX.Core
 {
 	public enum WeaponComponentHash : uint
 	{
-		AdvancedRifleClip01 = 4203716879u,
-		AdvancedRifleClip02 = 2395064697u,
-		AdvancedRifleVarmodLuxe = 930927479u,
```

Comment:
Removal and re-addition later, plus it seems this whole list's ordering is being messed up